paparazzi/build.gradle

```diff
@@ -95,6 +95,29 @@ subprojects {
       ])
     }
   }
+
+  // Avoid the native projects in 'libs'
+  plugins.withId('org.jetbrains.kotlin.jvm') {
+    // Create a test task for each supported JDK.
+    [11, 12, 16, 18].each { majorVersion ->
+      def jdkTest = tasks.register("testJdk$majorVersion", Test) {
+        javaLauncher = javaToolchains.launcherFor {
+          languageVersion = JavaLanguageVersion.of(majorVersion)
```

Comment:
Re CI failure, these might come in handy:
https://github.com/actions/setup-java#install-multiple-jdks
https://github.com/gradle/gradle/issues/14903#issuecomment-943421254